src/rviz/visualization_frame.cpp

```diff
@@ -447,9 +447,14 @@ void VisualizationFrame::savePersistentSettings()
 void VisualizationFrame::initMenus()
 {
   file_menu_ = menuBar()->addMenu( "&File" );
-  file_menu_->addAction( "&Open Config", this, SLOT( onOpen() ), QKeySequence( "Ctrl+O" ));
-  file_menu_->addAction( "&Save Config", this, SLOT( onSave() ), QKeySequence( "Ctrl+S" ));
-  file_menu_->addAction( "Save Config &As", this, SLOT( onSaveAs() ));
+
+  QAction * file_menu_open_action = file_menu_->addAction( "&Open Config", this, SLOT( onOpen() ), QKeySequence( "Ctrl+O" ));
+  this->addAction(file_menu_open_action);
+  QAction * file_menu_save_action = file_menu_->addAction( "&Save Config", this, SLOT( onSave() ), QKeySequence( "Ctrl+S" ));
+  this->addAction(file_menu_save_action);
+  QAction * file_menu_save_as_action = file_menu_->addAction( "Save Config &As", this, SLOT( onSaveAs() ), QKeySequence( "Ctrl+Shift+S"));
+  this->addAction(file_menu_save_as_action);
```

Comment:
That's interesting that this fixes it for you. Can you elaborate on why this is necessary? The Qt examples don't do that, but then again it could be related to the relationship between the UI elements in this particular scenario.

http://doc.qt.io/qt-5/qtwidgets-mainwindows-menus-example.html (I was looking at the section `void MainWindow::createMenus()`)

Maybe the `createMenus()` function is being overloaded and all we need to do is to change the name of this function.

I'm not certain about any of those ideas, I'm just throwing things out there incase you don't have a good explanation for why this fixes the problem and is the "correct" way to do it.